src/connectivity/rest/rest_controller.py

```diff
@@ -5,40 +5,44 @@
 from flask_restful import Resource, Api
 
 from configuration import Configuration as configuration
+from src.common_utils.database.collection_utils import parse_documents
 from src.common_utils.database.database_service import DatabaseProxy
 
 app = Flask(__name__)
 log = logging.getLogger('werkzeug')
 log.disabled = True
```

Comment:
is this needed in rest_controller since log.disabled = True?

Reply:
what do you mean exactly?

Reply:
can we remove these lines?